nsist/__init__.py

```diff
@@ -197,6 +200,15 @@ def fetch_python_embeddable(self):
         with zipfile.ZipFile(str(cache_file)) as z:
             z.extractall(python_dir)
 
+        # Manipulate any *._pth files so the default paths AND pkgs directory
+        # ends up in sys.path.
```

Comment:
Let's have a link in the comment to the docs describing this file: https://docs.python.org/3/using/windows.html#finding-modules